lib/livebook_app.ex

```diff
@@ -1,96 +1,103 @@
 if Mix.target() == :app do
-  defmodule LivebookApp do
+  defmodule WxConstants do
     @moduledoc false
 
-    @behaviour :wx_object
+    defmacro wxID_ANY, do: -1
+    defmacro wxID_OPEN, do: 5000
+    defmacro wxID_EXIT, do: 5006
+    defmacro wxID_OSX_HIDE, do: 5250
+    defmacro wxBITMAP_TYPE_PNG, do: 15
```

Comment:
Module attributes perhaps?

Reply:
I stuck with macros for two reasons: first, I can easily share it between modules; second, I can use it in function clauses. I moved it to the Utils module at the top of the file.